admin/class-yoast-dashboard-widget.php

```diff
@@ -110,48 +110,50 @@ private function set_statistic_items_for_this_user( $transient ) {
 	 * @return array
 	 */
 	private function get_seo_scores_with_post_count() {
+		$ranks = WPSEO_Rank::get_all_ranks();
+
+		return array_map( array( $this, 'map_rank_to_widget' ), $ranks );
+	}
+
+	/**
+	 * Converts a rank to data usable in the dashboard widget
+	 *
+	 * @param WPSEO_Rank $rank The rank to map.
+	 *
+	 * @return array
+	 */
+	private function map_rank_to_widget( $rank ) {
```

Comment:
I would typehint the parameter.

`private function map_rank_to_widget( WPSEO_Rank  $rank ) {`